### PR TITLE
Add update email dumb screen

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers

### CODE REVIEW

File: shared/settings/email/index.desktop.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+function UpdateEmail (props: Props) {
+  return (
+    <Box style={globalStyles.flexBoxColumn}>
+      <Input
+        floatingLabelText='Email'
+        value={props.email}
+        onChangeText={props.onChangeEmail}
+        textStyle={{height: undefined}} />
```
Comment (chromakode) [inline shared/settings/email/index.desktop.js]: Curious, why is this line needed?

**Comment by chromakode** [comment]
:+1:

File: shared/settings/email/index.desktop.js (hunk 1 above)
Comment (MarcoPolo) [inline shared/settings/email/index.desktop.js]: it probably isn't needed anymore, but at one point text had a set height and it made aligning things harder. Let me check now

Comment (MarcoPolo) [inline shared/settings/email/index.desktop.js]: Hmm, nope still has the set height. This is still needed.

The set height was to handle having error text below the input.

Comment (chromakode) [inline shared/settings/email/index.desktop.js]: Ah, thought so. Cool!
